### PR TITLE
Update examples using modern JS.

### PR DESCRIPTION
**Pull request by jan-ivar**
Fixes https://github.com/w3c/mediacapture-image/issues/252 and users `async`/`await` throughout.

### CODE REVIEW

**Comment by jan-ivar** [comment]
[split ignore-whitespace diff](https://github.com/w3c/mediacapture-image/pull/253/files?diff=split&w=1)

File: index.bs [hunk 1]
```diff
@@ -989,36 +969,34 @@ A {{Point2D}} represents a location in a two dimensional space. The origin of co
...
+          const canvas = document.querySelector('canvas');
+          const video = document.querySelector('video');
+
+          const stream = await navigator.mediaDevices.getUserMedia({video: true});
+          video.srcObject = stream;
+          const [track] = stream.getVideoTracks();
+          try {
+            const imageCapture = new ImageCapture(track);
+
+            stopButton.onclick = () => track.stop();
```
Comment (eehakkin) [inline index.bs:985]: Shouldn't this stopButton variable be defined before use?

**Review by eehakkin — CHANGES_REQUESTED** [review]
LGTM with nit.

File: index.bs (hunk 1 above)
Comment (beaufortfrancois) [reply index.bs:985]: `id="stopButton"` is enough in this case.

Comment (eehakkin) [reply index.bs:985]: Oh, I did not know that.

**Review by eehakkin — APPROVED** [review]
LGTM.

**Review by riju — APPROVED** [review]
Thanks @jan-ivar for modernizing the examples. I see that the `advanced` constraints were removed rightly as this was not needed in this case. We can check in https://github.com/w3c/mediacapture-image/issues/252 for that.

**Comment by beaufortfrancois** [comment]
I would not remove the `advanced` constraints in the applyConstraints examples as Chrome the only browser implementing it for now requires advanced constraints for using imageCapture constraints.
See https://source.chromium.org/chromium/chromium/src/+/master:third_party/blink/renderer/modules/mediastream/media_stream_track.cc;l=720;drc=ce68dbf2ceede2e29df8ec69eba5fadd870704f4;bpv=1;bpt=1?q=applyConstraints%20f:media&ss=chromium

**Comment by jan-ivar** [comment]
@beaufortfrancois Thanks for pointing that out. I've filed [crbug 1126045](https://bugs.chromium.org/p/chromium/issues/detail?id=1126045) on Chrome to update this to follow the spec.

**Pull request merged**